src/Microsoft.DotNet.Interactive/Connection/ProxyKernel.cs

```diff
@@ -121,6 +123,25 @@ private Task HandleByForwardingToRemoteAsync(KernelCommand command, KernelInvoca
         });
     }
 
+    private void EnsureKernelInfoIsLoaded(KernelCommand command, KernelInvocationContext context)
+    {
+        if (_requiresRequestKernelInfoOnFirstCommand)
```

Comment:
Can this be done using a deferred command?

Reply:
Though I guess that would produce a doubled `RequestKernelInfo` in some cases...